crates/turbopack-ecmascript-runtime/js/src/dev/runtime/nodejs/runtime-backend-nodejs.ts

```diff
@@ -28,6 +28,37 @@ function augmentContext(context: TurbopackDevBaseContext): TurbopackDevContext {
   return nodejsContext;
 }
 
+function resolveChunkPath(chunkPath: ChunkPath, source: SourceInfo) {
+  let fromChunkPath = undefined;
+  switch (source.type) {
+    case SourceType.Runtime:
+      fromChunkPath = source.chunkPath;
+      break;
+    case SourceType.Parent:
+      fromChunkPath = getFirstModuleChunk(source.parentId);
+      break;
+    case SourceType.Update:
+      break;
+  }
+
+  const path = require("path");
+  const resolved = require.resolve(
+    "./" + path.relative(path.dirname(fromChunkPath), chunkPath)
+  );
+
+  return resolved;
```

Comment:
`path.relative` won't work as expected on windows.

```suggestion
  return `${fromChunkPath}/../${chunkPath}`
```

Reply:
So the existing logic for loading chunks was also broken?